settings.py

```diff
@@ -49,6 +56,12 @@
 # or leave blank to skip gpg signing
 # CERT_KEY_ID = ''
 
+# Certificates Generation Directory
+TMP_GEN_DIR = '/tmp/certificates/'
+
+QUEUE_NAME = 'test-pull'
+QUEUE_URL = 'https://stage-xqueue.edx.org'
```

Comment:
I know this was already here, but we should definitely not be defaulting to a potentially real/existing URL.

Let's change this to `https://xqueue.example.com`.